### PR TITLE
Created Demo Test Cases + Removed Randomness

### PR DESCRIPTION
**Pull request by AndrejSafundzic**
1. Removed Randomness. The number nodes can be calculated in the following way: 
number of nodes = nodes_per_type * (num_types - 2)

2. Added a demo test cases called bootstrap_demo() looking like this:
WhatsApp Image 2020-06-07 at 10 26 26 PM

### CODE REVIEW

**Comment by nishantrai18** [comment]
Can we add a more complex base case for the demo? This is too small. Something with 6-7 stages at least -> 20-30ish nodes

**Comment by nishantrai18** [comment]
Umm this is not exactly what we meant by removing randomness. Can you take it out from here, I'll modify it in my PR.

File: src/basecases.py [hunk 1]
```diff
@@ -158,7 +218,7 @@ def bootstrap_random_dag(type_num=4, complexity="low", nodes_per_type=2):
             demo_nodes.append(node_tmp)
 
         else:
-            for j in range(1, random.randint(2, nodes_per_type)):
```
Comment (nishantrai18) [inline src/basecases.py:161]: Actually keep this part for now. We're short on time, so it's okay to have this in,

Comment (nishantrai18) [reply src/basecases.py:161]: Add random.seed() in random_dag(nodes, edges) to make the dag creation somewhat fixed.

Comment (AndrejSafundzic) [reply src/basecases.py:161]: DONE!

**Pull request merged**